vapi/tags/categories.go

```diff
@@ -108,13 +111,20 @@ func (c *Manager) UpdateCategory(ctx context.Context, category *Category) error
 		},
 	}
 	url := c.Resource(internal.CategoryPath).WithID(category.ID)
-	return c.Do(ctx, url.Request(http.MethodPatch, spec), nil)
+	req := url.Request(http.MethodPatch, spec)
+	if err := c.Do(ctx, req, nil); err != nil {
+		return err
+	}
+	return nil
 }
 
 // DeleteCategory deletes a category.
 func (c *Manager) DeleteCategory(ctx context.Context, category *Category) error {
 	url := c.Resource(internal.CategoryPath).WithID(category.ID)
-	return c.Do(ctx, url.Request(http.MethodDelete), nil)
```

Comment:
Same, just returning an error

Reply:
Resolved in https://github.com/vmware/govmomi/commit/d14cbf39ee092c1a12388ab5b8009bc3c443c898.